common.js

```diff
@@ -4,7 +4,7 @@ var config = require('./config'),
 function getFormattedDate(date, date_format) {
     date_format = date_format || config.date_format;
 
-    return moment(date).format(date_format);
+    return moment(date, date_format);
```

Comment:
no additional parameter but it should be ISO 8601 format which is expected by jsonresume schema